src/modules/dex/directives/createOrder/CreateOrder.js

```diff
@@ -254,6 +254,7 @@
                 if (data) {
                     this.amountBalance = data.amountBalance;
                     this.priceBalance = data.priceBalance;
+                    this.maxAmountBalance = CreateOrder._getMaxAmountBalance(this.type, this.amountBalance, this.fee);
```

Comment:
Taking into account that exactly the same expression is on line 134 it is better to extract it to a method and use both here and there.